2.JSX.md

```diff
@@ -0,0 +1,34 @@
+**Jam3 Lessons - React**
+
+# JSX
+
+First of all, JSX is **neither** a string **nor** HTML.
+
+> Fundamentally, JSX just provides syntactic sugar for the `React.createElement(component, props, ...children)` function.
+
+```JSX
+<Welcome size="huge">
+  Hey, you!
+</Welcome>
+```
+
+compiles into
+
+```JSX
+React.createElement(
+  Welcome,
+  { size: 'huge' },
+  'Hey, you!'
+);
+```
+
+### Use Capitalized names for user-defined components
+
+`React` thinks elements starting with a lowercase letter are native HTML elements. And elements starting with a capital letter are considered and corresponded to the components the user defined.
```

Comment:
I would erase the period `.` and write it like `HTML elements and elements with a...`, what do you think?